keyboards/40percentclub/gherkin/keymaps/pierrec83/keymap.c

```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file should be deleted, the build process will automatically regenerate it.

Reply:
Oh! That is a great piece of information :)
Like you said, `qmk flash` still works without this extra step of generating the keymap manually. That's really cool!